packages/account/src/Sections/Profile/PhoneVerification/__test__/validation.spec.tsx

```diff
@@ -4,53 +4,140 @@ import { validatePhoneNumber } from '../validation';
 describe('validatePhoneNumber', () => {
     const setErrorMessage = jest.fn();
     const setIsDisabledRequestButton = jest.fn();
+    let isCountryCodeDropdownEnabled = false;
 
     const error_message = 'Enter a valid phone number.';
 
     it('should set an empty error message for a valid phone number', async () => {
         const validPhoneNumber = '+1234567890';
         await act(async () => {
-            validatePhoneNumber(validPhoneNumber, setErrorMessage, setIsDisabledRequestButton);
+            validatePhoneNumber(
+                validPhoneNumber,
+                setErrorMessage,
+                setIsDisabledRequestButton,
+                isCountryCodeDropdownEnabled
+            );
         });
         expect(setErrorMessage).toHaveBeenCalledWith('');
     });
 
     it('should set an error message for an invalid phone number', async () => {
         const invalidPhoneNumber = 'invalid';
         await act(async () => {
-            validatePhoneNumber(invalidPhoneNumber, setErrorMessage, setIsDisabledRequestButton);
+            validatePhoneNumber(
+                invalidPhoneNumber,
+                setErrorMessage,
+                setIsDisabledRequestButton,
+                isCountryCodeDropdownEnabled
+            );
         });
         expect(setErrorMessage).toHaveBeenCalledWith([error_message]);
     });
 
     it('should set an error message for an empty phone number', async () => {
         const invalidPhoneNumber = '';
         await act(async () => {
-            validatePhoneNumber(invalidPhoneNumber, setErrorMessage, setIsDisabledRequestButton);
+            validatePhoneNumber(
+                invalidPhoneNumber,
+                setErrorMessage,
+                setIsDisabledRequestButton,
+                isCountryCodeDropdownEnabled
+            );
         });
         expect(setErrorMessage).toHaveBeenCalledWith([error_message]);
     });
 
     it('should set an error message for an phone number more than 36 characters', async () => {
         const invalidPhoneNumber = '+123123123123123123123123232333333333';
         await act(async () => {
-            validatePhoneNumber(invalidPhoneNumber, setErrorMessage, setIsDisabledRequestButton);
+            validatePhoneNumber(
+                invalidPhoneNumber,
+                setErrorMessage,
+                setIsDisabledRequestButton,
+                isCountryCodeDropdownEnabled
+            );
         });
         expect(setErrorMessage).toHaveBeenCalledWith([error_message]);
     });
 
     it('should set an error message for an phone number less than 8 characters', async () => {
         const invalidPhoneNumber = '+1234567';
         await act(async () => {
-            validatePhoneNumber(invalidPhoneNumber, setErrorMessage, setIsDisabledRequestButton);
+            validatePhoneNumber(
```

Comment:
The callback function of `act` has async but`validatePhoneNumber` there is no await used in callback function. Do we need the async?

Reply:
Yes i've tested it, without `async` it will break the test case 🙏